1.Practicals_C/Pointers/Pointers_in_structures.c

```diff
@@ -0,0 +1,22 @@
+  #include<stdio.h>
+   struct s 
+    { 
+      int i; 
+      struct s *p; /*pointer as a variable in 
+                     structure.It stores 
+                     address of the structure */
+    }; 
+   void main( ) 
+   { 
+        struct s var1,var2; 
+        var1.i=100; 
+        var2.i=200; 
+        var1.p=&var2; /*here address of var1 is 
+        var2.p=&var1;   stored in pointer of 
```

Comment:
Write the comment and code in different line, otherwise they are not executed properly.
```suggestion
        var1.p=&var2;
        var2.p=&var1;  /*comment*/
```